src/components/Auth/Auth.jsx

```diff
@@ -10,20 +10,20 @@ export default class Auth extends React.Component {
       authenticated: false
     };
   }
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
`componentWillMount` is equivalent to the constructor. I think the general consensus is that this kind of thing would be called in `componentDidMount` but it doesn't really matter since API calls are asynchronus. However, will the API calls be moving into actions at some point?

Reply:
I changed it to componentDidMount in the other PR... I think since we're not storing any state with the logging right now (the cookie is just saved in the browser), we don't need to move the authorisation requests to the actions for now (I spoke to Alex about this and that's what he told me)

Reply:
OK, fair enough. I'm not sure I agree though, it would make sense that `loading` is a global app state and `authenticated` definitely seems like something that should be available outside of this component.

Reply:
That's true, I could create an issue for it and we could fix it if we have time in this sprint?

Reply:
It's something that maybe warrants a discussion but yeah, don't worry about it for now.